packages/reports/src/Containers/open-positions.tsx

```diff
@@ -190,23 +191,29 @@ const OpenPositions = observer(({ component_icon, ...props }: TOpenPositions) =>
     const { isDesktop } = useDevice();
     const previous_active_positions = usePrevious(active_positions);
     const contract_types = [
-        { text: localize('Options'), is_default: !is_multiplier && !is_accumulator },
-        { text: localize('Multipliers'), is_default: is_multiplier },
-        { text: localize('Accumulators'), is_default: is_accumulator },
+        { text: localize('Options'), value: 'Options', is_default: !is_multiplier && !is_accumulator },
```

Comment:
Not a huge issue but it's better if you make values low-case.

Reply:
Okay, will apply